src/Twig/ThemeFilesystemLoader.php

```diff
@@ -60,7 +60,7 @@ public function getSourceContext($name): \Twig_Source
     public function getCacheKey($name): string
     {
         try {
-            return $this->findTemplate($name);
+            return $this->findTemplate((string) $name);
```

Comment:
Could we put casting to string inside `findTemplate` method? That method is also used in `isFresh` and `exists`.

Reply:
Gonna fix it in a separate PR